geopandas/geodataframe.py

```diff
@@ -431,19 +424,14 @@ def crs(self):
         GeoDataFrame.to_crs : re-project to another CRS
 
         """
-        # TODO: remove try/except in 0.12
         try:
             return self.geometry.crs
         except AttributeError:
-            # the active geometry column might not be set
-            warnings.warn(
-                "Accessing CRS of a GeoDataFrame without a geometry column is "
-                "deprecated and will be removed in GeoPandas 0.12. "
-                "Use GeoDataFrame.set_geometry to set the active geometry column.",
-                FutureWarning,
-                stacklevel=2,
+            raise AttributeError(
```

Comment:
Can we include `stacklevel` here? Guessing from the existing code, `stacklevel=2`.

Reply:
I don't think so, no. As far as I am aware python exceptions don't support stacklevel.

Reply:
Yeah, I didn't realise we're changing warning to exception when writing that... traceback resolves that.